pages/linux/dnsenum.md

```diff
@@ -0,0 +1,16 @@
+# dnsenum
+
+> Multithreaded Perl script used to enumerate DNS information from a domain.
+> More information: <https://github.com/SparrowOchon/dnsenum2>.
+
+- Enumerate a domain with default settings:
+
+`dnsenum -enum {{example.com}}`
+
+- Enumerate a domain without reverse lookup and save the output to a file:
+
+`dnsenum --noreverse -o {{path/to/file}} {{example.com}}`
```

Comment:
```suggestion
- Enumerate a domain without reverse lookup and save the [o]utput to a specific file:

`dnsenum --noreverse -o {{path/to/file}} {{example.com}}`
```

Almost the same issue as above.
Warning: [parsing man page failed, can't find `-o` option, possibly broken manual](https://manned.org/dnsenum.1).